### ISSUE TITLE
Error Encountered While Running HibiAPI Docker Image

### ISSUE THREAD

**Issue by pppaapp** [mixmoe/HibiAPI#401]
hello

I build the HibiAPI Docker image today (2024/06/10) and encountered the following error logs when running the image:

```
 WARNING  [2024/06/10 02:41:04.977 __main__:hibiapi.__main__:81] 
  _    _ _ _     _          _____ _____  
 | |  | (_) |   (_)   /\   |  __ \_   _| 
 | |__| |_| |__  _   /  \  | |__) || |   
 |  __  | | '_ \| | / /\ \ |  ___/ | |   
 | |  | | | |_) | |/ ____ \| |    _| |_  
 |_|  |_|_|_.__/|_/_/    \_\_|   |_____| 

A program that implements easy-to-use APIs for a variety of commonly used sites
Repository: https://github.com/mixmoe/HibiAPI

 INFO     [2024/06/10 02:41:04.977 __main__:hibiapi.__main__:84] HibiAPI version: 0.8.0
╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /usr/local/lib/python3.12/site-packages/hibiapi/__main__.py:86 in run        │
│                                                                              │
│    83 │   )                                                                  │
│    84 │   logger.info(f"HibiAPI version: <g><b>{__version__}</b></g>")       │
│    85 │                                                                      │
│ ❱  86 │   uvicorn.run(                                                       │
│    87 │   │   "hibiapi.app:app",                                             │
│    88 │   │   host=host,                                                     │
│    89 │   │   port=port,                                                     │
│                                                                              │
│ ╭──────────────────────────── locals ────────────────────────────╮           │
│ │            ctx = <click.core.Context object at 0xffff4ab2bdd0> │           │
│ │      func_name = 'run'                                         │           │
│ │           host = '0.0.0.0'                                     │           │
│ │           port = 8080                                          │           │
│ │         reload = False                                         │           │
│ │ terminal_width = 0                                             │           │
│ │        workers = 1                                             │           │
│ ╰────────────────────────────────────────────────────────────────╯           │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/uvicorn/main.py:569 in run           │
│                                                                              │
│   566 │   │   sock = config.bind_socket()                                    │
│   567 │   │   Multiprocess(config, target=server.run, sockets=[sock]).run()  │
│   568 │   else:                                                              │
│ ❱ 569 │   │   server.run()                                                   │
│   570 │   if config.uds:                                                     │
│   571 │   │   os.remove(config.uds)  # pragma: py-win32                      │
│   572                                                                        │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │                    access_log = False                                    │ │
│ │                           app = 'hibiapi.app:app'                        │ │
│ │                       app_dir = None                                     │ │
│ │                       backlog = 2048                                     │ │
│ │                        config = <uvicorn.config.Config object at         │ │
│ │                                 0xffff4b9f8cb0>                          │ │
│ │                   date_header = True                                     │ │
│ │                      env_file = None                                     │ │
│ │                       factory = False                                    │ │
│ │                            fd = None                                     │ │
│ │           forwarded_allow_ips = None                                     │ │
│ │ h11_max_incomplete_event_size = 16384                                    │ │
│ │                       headers = None                                     │ │
│ │                          host = '0.0.0.0'                                │ │
│ │                          http = 'auto'                                   │ │
│ │                     interface = 'auto'                                   │ │
│ │                      lifespan = 'auto'                                   │ │
│ │             limit_concurrency = None                                     │ │
│ │            limit_max_requests = None                                     │ │
│ │                    log_config = {                                        │ │
│ │                                 │   'version': 1,                        │ │
│ │                                 │   'disable_existing_loggers': False,   │ │
│ │                                 │   'handlers': {                        │ │
│ │                                 │   │   'default': {                     │ │
│ │                                 │   │   │   'class':                     │ │
│ │                                 'hibiapi.utils.log.LoguruHandler'        │ │
│ │                                 │   │   }                                │ │
│ │                                 │   },                                   │ │
│ │                                 │   'loggers': {                         │ │
│ │                                 │   │   'uvicorn.error': {               │ │
│ │                                 │   │   │   'handlers': ['default'],     │ │
│ │                                 │   │   │   'level': 'INFO'              │ │
│ │                                 │   │   },                               │ │
│ │                                 │   │   'uvicorn.access': {              │ │
│ │                                 │   │   │   'handlers': ['default'],     │ │
│ │                                 │   │   │   'level': 'INFO'              │ │
│ │                                 │   │   }                                │ │
│ │                                 │   }                                    │ │
│ │                                 }                                        │ │
│ │                     log_level = None                                     │ │
│ │                          loop = 'auto'                                   │ │
│ │                          port = 8080                                     │ │
│ │                 proxy_headers = True                                     │ │
│ │                        reload = False                                    │ │
│ │                  reload_delay = 0.25                                     │ │
│ │                   reload_dirs = None                                     │ │
│ │               reload_excludes = None                                     │ │
│ │               reload_includes = None                                     │ │
│ │                     root_path = ''                                       │ │
│ │                        server = <uvicorn.server.Server object at         │ │
│ │                                 0xffff4b250d70>                          │ │
│ │                 server_header = True                                     │ │
│ │                  ssl_ca_certs = None                                     │ │
│ │                 ssl_cert_reqs = <VerifyMode.CERT_NONE: 0>                │ │
│ │                  ssl_certfile = None                                     │ │
│ │                   ssl_ciphers = 'TLSv1'                                  │ │
│ │                   ssl_keyfile = None                                     │ │
│ │          ssl_keyfile_password = None                                     │ │
│ │                   ssl_version = <_SSLMethod.PROTOCOL_TLS_SERVER: 17>     │ │
│ │            timeout_keep_alive = 5                                        │ │
│ │                           uds = None                                     │ │
│ │                    use_colors = None                                     │ │
│ │                       workers = 1                                        │ │
│ │                            ws = 'auto'                                   │ │
│ │                   ws_max_size = 16777216                                 │ │
│ │        ws_per_message_deflate = True                                     │ │
│ │              ws_ping_interval = 20.0                                     │ │
│ │               ws_ping_timeout = 20.0                                     │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/uvicorn/server.py:60 in run          │
│                                                                              │
│    57 │                                                                      │
│    58 │   def run(self, sockets: Optional[List[socket.socket]] = None) -> No │
│    59 │   │   self.config.setup_event_loop()                                 │
│ ❱  60 │   │   return asyncio.run(self.serve(sockets=sockets))                │
│    61 │                                                                      │
│    62 │   async def serve(self, sockets: Optional[List[socket.socket]] = Non │
│    63 │   │   process_id = os.getpid()                                       │
│                                                                              │
│ ╭────────────────────────── locals ──────────────────────────╮               │
│ │    self = <uvicorn.server.Server object at 0xffff4b250d70> │               │
│ │ sockets = None                                             │               │
│ ╰────────────────────────────────────────────────────────────╯               │
│                                                                              │
│ /usr/local/lib/python3.12/asyncio/runners.py:194 in run                      │
│                                                                              │
│   191 │   │   │   "asyncio.run() cannot be called from a running event loop" │
│   192 │                                                                      │
│   193 │   with Runner(debug=debug, loop_factory=loop_factory) as runner:     │
│ ❱ 194 │   │   return runner.run(main)                                        │
│   195                                                                        │
│   196                                                                        │
│   197 def _cancel_all_tasks(loop):                                           │
│                                                                              │
│ ╭───────────────────────────── locals ─────────────────────────────╮         │
│ │        debug = None                                              │         │
│ │ loop_factory = None                                              │         │
│ │         main = <coroutine object Server.serve at 0xffff4ab60340> │         │
│ │       runner = <asyncio.runners.Runner object at 0xffff4ae291f0> │         │
│ ╰──────────────────────────────────────────────────────────────────╯         │
│                                                                              │
│ /usr/local/lib/python3.12/asyncio/runners.py:118 in run                      │
│                                                                              │
│   115 │   │                                                                  │
│   116 │   │   self._interrupt_count = 0                                      │
│   117 │   │   try:                                                           │
│ ❱ 118 │   │   │   return self._loop.run_until_complete(task)                 │
│   119 │   │   except exceptions.CancelledError:                              │
│   120 │   │   │   if self._interrupt_count > 0:                              │
│   121 │   │   │   │   uncancel = getattr(task, "uncancel", None)             │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │        context = <_contextvars.Context object at 0xffff4ad386c0>         │ │
│ │           coro = <coroutine object Server.serve at 0xffff4ab60340>       │ │
│ │           self = <asyncio.runners.Runner object at 0xffff4ae291f0>       │ │
│ │ sigint_handler = functools.partial(<bound method Runner._on_sigint of    │ │
│ │                  <asyncio.runners.Runner object at 0xffff4ae291f0>>,     │ │
│ │                  main_task=<Task finished name='Task-1'                  │ │
│ │                  coro=<Server.serve() done, defined at                   │ │
│ │                  /usr/local/lib/python3.12/site-packages/uvicorn/server… │ │
│ │                  exception=TypeError("ForwardRef._evaluate() missing 1   │ │
│ │                  required keyword-only argument: 'recursive_guard'")>)   │ │
│ │           task = <Task finished name='Task-1' coro=<Server.serve() done, │ │
│ │                  defined at                                              │ │
│ │                  /usr/local/lib/python3.12/site-packages/uvicorn/server… │ │
│ │                  exception=TypeError("ForwardRef._evaluate() missing 1   │ │
│ │                  required keyword-only argument: 'recursive_guard'")>    │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ in uvloop.loop.Loop.run_until_complete:1517                                  │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/uvicorn/server.py:67 in serve        │
│                                                                              │
│    64 │   │                                                                  │
│    65 │   │   config = self.config                                           │
│    66 │   │   if not config.loaded:                                          │
│ ❱  67 │   │   │   config.load()                                              │
│    68 │   │                                                                  │
│    69 │   │   self.lifespan = config.lifespan_class(config)                  │
│    70                                                                        │
│                                                                              │
│ ╭─────────────────────────── locals ────────────────────────────╮            │
│ │     config = <uvicorn.config.Config object at 0xffff4b9f8cb0> │            │
│ │ process_id = 7                                                │            │
│ │       self = <uvicorn.server.Server object at 0xffff4b250d70> │            │
│ │    sockets = None                                             │            │
│ ╰───────────────────────────────────────────────────────────────╯            │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/uvicorn/config.py:474 in load        │
│                                                                              │
│   471 │   │   self.lifespan_class = import_from_string(LIFESPAN[self.lifespa │
│   472 │   │                                                                  │
│   473 │   │   try:                                                           │
│ ❱ 474 │   │   │   self.loaded_app = import_from_string(self.app)             │
│   475 │   │   except ImportFromStringError as exc:                           │
│   476 │   │   │   logger.error("Error loading ASGI app. %s" % exc)           │
│   477 │   │   │   sys.exit(1)                                                │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │     encoded_headers = []                                                 │ │
│ │ http_protocol_class = <class                                             │ │
│ │                       'uvicorn.protocols.http.httptools_impl.HttpToolsP… │ │
│ │                self = <uvicorn.config.Config object at 0xffff4b9f8cb0>   │ │
│ │   ws_protocol_class = <class                                             │ │
│ │                       'uvicorn.protocols.websockets.websockets_impl.Web… │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/uvicorn/importer.py:21 in            │
│ import_from_string                                                           │
│                                                                              │
│   18 │   │   raise ImportFromStringError(message.format(import_str=import_st │
│   19 │                                                                       │
│   20 │   try:                                                                │
│ ❱ 21 │   │   module = importlib.import_module(module_str)                    │
│   22 │   except ImportError as exc:                                          │
│   23 │   │   if exc.name != module_str:                                      │
│   24 │   │   │   raise exc from None                                         │
│                                                                              │
│ ╭──────────── locals ────────────╮                                           │
│ │          _ = ':'               │                                           │
│ │  attrs_str = 'app'             │                                           │
│ │ import_str = 'hibiapi.app:app' │                                           │
│ │ module_str = 'hibiapi.app'     │                                           │
│ ╰────────────────────────────────╯                                           │
│                                                                              │
│ /usr/local/lib/python3.12/importlib/__init__.py:90 in import_module          │
│                                                                              │
│    87 │   │   │   if character != '.':                                       │
│    88 │   │   │   │   break                                                  │
│    89 │   │   │   level += 1                                                 │
│ ❱  90 │   return _bootstrap._gcd_import(name[level:], package, level)        │
│    91                                                                        │
│    92                                                                        │
│    93 _RELOADING = {}                                                        │
│                                                                              │
│ ╭──────── locals ─────────╮                                                  │
│ │   level = 0             │                                                  │
│ │    name = 'hibiapi.app' │                                                  │
│ │ package = None          │                                                  │
│ ╰─────────────────────────╯                                                  │
│ in _gcd_import:1387                                                          │
│ ╭──────── locals ─────────╮                                                  │
│ │   level = 0             │                                                  │
│ │    name = 'hibiapi.app' │                                                  │
│ │ package = None          │                                                  │
│ ╰─────────────────────────╯                                                  │
│ in _find_and_load:1360                                                       │
│ ╭────────────────────── locals ──────────────────────╮                       │
│ │ import_ = <function _gcd_import at 0xffff4c8a40e0> │                       │
│ │  module = <object object at 0xffff4c87c060>        │                       │
│ │    name = 'hibiapi.app'                            │                       │
│ ╰────────────────────────────────────────────────────╯                       │
│ in _find_and_load_unlocked:1331                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │         child = 'app'                                                    │ │
│ │       import_ = <function _gcd_import at 0xffff4c8a40e0>                 │ │
│ │          name = 'hibiapi.app'                                            │ │
│ │        parent = 'hibiapi'                                                │ │
│ │ parent_module = <module 'hibiapi' from                                   │ │
│ │                 '/usr/local/lib/python3.12/site-packages/hibiapi/__init… │ │
│ │   parent_spec = ModuleSpec(name='hibiapi',                               │ │
│ │                 loader=<_frozen_importlib_external.SourceFileLoader      │ │
│ │                 object at 0xffff4c73ba70>,                               │ │
│ │                 origin='/usr/local/lib/python3.12/site-packages/hibiapi… │ │
│ │                 submodule_search_locations=['/usr/local/lib/python3.12/… │ │
│ │          path = ['/usr/local/lib/python3.12/site-packages/hibiapi']      │ │
│ │          spec = ModuleSpec(name='hibiapi.app',                           │ │
│ │                 loader=<_frozen_importlib_external.SourceFileLoader      │ │
│ │                 object at 0xffff4a79b260>,                               │ │
│ │                 origin='/usr/local/lib/python3.12/site-packages/hibiapi… │ │
│ │                 submodule_search_locations=['/usr/local/lib/python3.12/… │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│ in _load_unlocked:935                                                        │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │ module = <module 'hibiapi.app' from                                      │ │
│ │          '/usr/local/lib/python3.12/site-packages/hibiapi/app/__init__.… │ │
│ │   spec = ModuleSpec(name='hibiapi.app',                                  │ │
│ │          loader=<_frozen_importlib_external.SourceFileLoader object at   │ │
│ │          0xffff4a79b260>,                                                │ │
│ │          origin='/usr/local/lib/python3.12/site-packages/hibiapi/app/__… │ │
│ │          submodule_search_locations=['/usr/local/lib/python3.12/site-pa… │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│ in exec_module:995                                                           │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │   code = <code object <module> at 0xffff4ab5ff00, file                   │ │
│ │          "/usr/local/lib/python3.12/site-packages/hibiapi/app/__init__.… │ │
│ │          line 1>                                                         │ │
│ │ module = <module 'hibiapi.app' from                                      │ │
│ │          '/usr/local/lib/python3.12/site-packages/hibiapi/app/__init__.… │ │
│ │   self = <_frozen_importlib_external.SourceFileLoader object at          │ │
│ │          0xffff4a79b260>                                                 │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│ in _call_with_frames_removed:488                                             │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │ args = (                                                                 │ │
│ │        │   <code object <module> at 0xffff4ab5ff00, file                 │ │
│ │        "/usr/local/lib/python3.12/site-packages/hibiapi/app/__init__.py… │ │
│ │        line 1>,                                                          │ │
│ │        │   {                                                             │ │
│ │        │   │   '__name__': 'hibiapi.app',                                │ │
│ │        │   │   '__doc__': None,                                          │ │
│ │        │   │   '__package__': 'hibiapi.app',                             │ │
│ │        │   │   '__loader__':                                             │ │
│ │        <_frozen_importlib_external.SourceFileLoader object at            │ │
│ │        0xffff4a79b260>,                                                  │ │
│ │        │   │   '__spec__': ModuleSpec(name='hibiapi.app',                │ │
│ │        loader=<_frozen_importlib_external.SourceFileLoader object at     │ │
│ │        0xffff4a79b260>,                                                  │ │
│ │        origin='/usr/local/lib/python3.12/site-packages/hibiapi/app/__in… │ │
│ │        submodule_search_locations=['/usr/local/lib/python3.12/site-pack… │ │
│ │        │   │   '__path__': [                                             │ │
│ │        │   │   │   '/usr/local/lib/python3.12/site-packages/hibiapi/app' │ │
│ │        │   │   ],                                                        │ │
│ │        │   │   '__file__':                                               │ │
│ │        '/usr/local/lib/python3.12/site-packages/hibiapi/app/__init__.py… │ │
│ │        │   │   '__cached__':                                             │ │
│ │        '/usr/local/lib/python3.12/site-packages/hibiapi/app/__pycache__… │ │
│ │        │   │   '__builtins__': {                                         │ │
│ │        │   │   │   '__name__': 'builtins',                               │ │
│ │        │   │   │   '__doc__': 'Built-in functions, types, exceptions,    │ │
│ │        and other objects.\n\nThis module provides '+346,                 │ │
│ │        │   │   │   '__package__': '',                                    │ │
│ │        │   │   │   '__loader__': <class                                  │ │
│ │        '_frozen_importlib.BuiltinImporter'>,                             │ │
│ │        │   │   │   '__spec__': ModuleSpec(name='builtins', loader=<class │ │
│ │        '_frozen_importlib.BuiltinImporter'>, origin='built-in'),         │ │
│ │        │   │   │   '__build_class__': <built-in function                 │ │
│ │        __build_class__>,                                                 │ │
│ │        │   │   │   '__import__': <built-in function __import__>,         │ │
│ │        │   │   │   'abs': <built-in function abs>,                       │ │
│ │        │   │   │   'all': <built-in function all>,                       │ │
│ │        │   │   │   'any': <built-in function any>,                       │ │
│ │        │   │   │   ... +147                                              │ │
│ │        │   │   }                                                         │ │
│ │        │   }                                                             │ │
│ │        )                                                                 │ │
│ │    f = <built-in function exec>                                          │ │
│ │ kwds = {}                                                                │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/hibiapi/app/__init__.py:2 in         │
│ <module>                                                                     │
│                                                                              │
│   1 # flake8:noqa:F401                                                       │
│ ❱ 2 from . import application, handlers, middlewares                         │
│   3                                                                          │
│   4 app = application.app                                                    │
│   5                                                                          │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/hibiapi/app/application.py:8 in      │
│ <module>                                                                     │
│                                                                              │
│     5 from urllib.parse import ParseResult                                   │
│     6                                                                        │
│     7 import sentry_sdk                                                      │
│ ❱   8 from fastapi import Depends, FastAPI, Request, Response                │
│     9 from fastapi.security import HTTPBasic, HTTPBasicCredentials           │
│    10 from fastapi.staticfiles import StaticFiles                            │
│    11 from pydantic import BaseModel                                         │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │        asyncio = <module 'asyncio' from                                  │ │
│ │                  '/usr/local/lib/python3.12/asyncio/__init__.py'>        │ │
│ │ compare_digest = <built-in function compare_digest>                      │ │
│ │     ip_address = <function ip_address at 0xffff4c77eca0>                 │ │
│ │           List = typing.List                                             │ │
│ │    ParseResult = <class 'urllib.parse.ParseResult'>                      │ │
│ │     sentry_sdk = <module 'sentry_sdk' from                               │ │
│ │                  '/usr/local/lib/python3.12/site-packages/sentry_sdk/__… │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/fastapi/__init__.py:7 in <module>    │
│                                                                              │
│    4                                                                         │
│    5 from starlette import status as status                                  │
│    6                                                                         │
│ ❱  7 from .applications import FastAPI as FastAPI                            │
│    8 from .background import BackgroundTasks as BackgroundTasks              │
│    9 from .datastructures import UploadFile as UploadFile                    │
│   10 from .exceptions import HTTPException as HTTPException                  │
│                                                                              │
│ ╭───────────────────────────────── locals ─────────────────────────────────╮ │
│ │    _compat = <module 'fastapi._compat' from                              │ │
│ │              '/usr/local/lib/python3.12/site-packages/fastapi/_compat.p… │ │
│ │ exceptions = <module 'fastapi.exceptions' from                           │ │
│ │              '/usr/local/lib/python3.12/site-packages/fastapi/exception… │ │
│ │     logger = <module 'fastapi.logger' from                               │ │
│ │              '/usr/local/lib/python3.12/site-packages/fastapi/logger.py… │ │
│ │    openapi = <module 'fastapi.openapi' from                              │ │
│ │              '/usr/local/lib/python3.12/site-packages/fastapi/openapi/_… │ │
│ │     status = <module 'starlette.status' from                             │ │
│ │              '/usr/local/lib/python3.12/site-packages/starlette/status.… │ │
│ │      types = <module 'fastapi.types' from                                │ │
│ │              '/usr/local/lib/python3.12/site-packages/fastapi/types.py'> │ │
│ ╰──────────────────────────────────────────────────────────────────────────╯ │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/fastapi/applications.py:16 in        │
│ <module>                                                                     │
│                                                                              │
│     13 │   Union,                                                            │
│     14 )                                                                     │
│     15                                                                       │
│ ❱   16 from fastapi import routing                                           │
│     17 from fastapi.datastructures import Default, DefaultPlaceholder        │
│     18 from fastapi.exception_handlers import (                              │
│     19 │   http_exception_handler,                                           │
│                                                                              │
│ ╭─────────────── locals ───────────────╮                                     │
│ │       Any = typing.Any               │                                     │
│ │ Awaitable = typing.Awaitable         │                                     │
│ │  Callable = typing.Callable          │                                     │
│ │ Coroutine = typing.Coroutine         │                                     │
│ │      Dict = typing.Dict              │                                     │
│ │      Enum = <enum 'Enum'>            │                                     │
│ │      List = typing.List              │                                     │
│ │  Optional = typing.Optional          │                                     │
│ │  Sequence = typing.Sequence          │                                     │
│ │      Type = typing.Type              │                                     │
│ │   TypeVar = <class 'typing.TypeVar'> │                                     │
│ │     Union = typing.Union             │                                     │
│ ╰──────────────────────────────────────╯                                     │
│                                                                              │
│ /usr/local/lib/python3.12/site-packages/fastapi/routing.py:22 in <module>    │
│                                                                              │
│     19 │   Union,                                                            │
│     20 )                                                                     │
│     21                                                                       │
│ ❱   22 from fastapi import params                                            │
│     23 from fastapi._compat import (                                         │
│     24 │   ModelField,                                                       │
│     25 │   Undefined,                                                        │
│                                                                              │
│ ╭───────────────────────────────── locals 

...
```

[1.log](https://github.com/user-attachments/files/15755870/1.log)

**Comment by mnixry** [comment]
It appears this is an ongoing issue with PyDantic, possibly even CPython. Refer to https://github.com/pydantic/pydantic/issues/9609 for more details.

TLDR, you might need to revert your Python version back to 3.12.3 or earlier.

**Comment by pppaapp** [comment]
The issue is resolved
